### DISCUSSION TITLE
Cannot access Image.tag on the server. `next/image` and `react-dom/server` are not playing nice together

### DISCUSSION THREAD

**Discussion by JacobWeisenburger** [vercel/next.js#58456] [Help]
When I do this:

```tsx
import Image from 'next/image'
import FooImage from '../../../public/images/foo.webp'

export default async function Page () {
    const { renderToString } = await import( 'react-dom/server' )
    const fooHTML = renderToString( <Image src={FooImage} alt={'Foo'} /> ) // throws

    return <main>
        <div dangerouslySetInnerHTML={{ __html: fooHTML }}></div>
        <Image src={FooImage} alt={'Foo'} /> {/* works */}
    </main>
}
```

I get this error

`
Error: Cannot access Image.tag on the server. You cannot dot into a client module from a server component. You can only pass the imported name through.
`

![image](https://github.com/vercel/next.js/assets/31667350/05106349-7c2a-4256-9b4d-84ce23edb658)

What am I doing wrong?

**Comment by isaacdarcilla** [comment]
Can you try this?

```
import FooImage from '/images/a-couple-holding-a-brochure-for-a-tourist-attraction-400.webp'
```

**Reply by JacobWeisenburger** [reply]
Thanks for trying, but that didn't help. I have updated the original post to better show the problem. Seems to have something to do with `renderToString` or `react-dom/server`

**Reply by numito** [reply]
next/image is a client component, and therefore cannot be rendered with renderToString as it needs to add javascript and hooks.
